src/components/Link/Link.tsx

```diff
@@ -3,7 +3,7 @@
 import { AnchorHTMLAttributes, ForwardedRef, forwardRef, ReactNode, useState, useEffect } from 'react';
 import { NavLink, Link as RouterLink, To } from 'react-router-dom';
 
-export type LinkVariants = '' | 'cta' | 'ghost' | 'mailto' | 'nav' | 'subtle';
+export type LinkVariants = 'cta' | 'download-button' | 'ghost' | 'mailto' | 'nav' | 'subtle';
```

Comment:
These link variants only affect styles. Since this DownloadLink shares so much (everything style-wise) with the regular link, I think this is the way to go.